src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedReference.cs

```diff
@@ -27,7 +28,8 @@ protected EncryptedReference(string uri, TransformChain transformChain)
             _cachedXml = null;
         }
 
-        public string Uri
+        [DisallowNull]
+        public string? Uri
```

Comment:
I don't see how _uri becomes null.  Shouldn't this just be `public string Uri` with no additional attributes?